modules/microshift-restoring-backups.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_backup_and_restore/microshift-auto-recover-manual-backup.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-restoring-backups_{context}"]
+= Restoring backups using the auto-recovery feature
+
+You can restore backups after system events that remove or damage required data.
+
+You can use the following procedure to restore backups.
+
+.Prerequisites
+
+* You have stopped {microshift-short}.
+
+.Procedure
+
+. Restore the backup from the directory in which you have stored the backups by running the following command:
++
+[source,terminal]
+----
+$ sudo microshift restore --auto-recovery _<path_of_directory>_ <1>
```

Comment:
Same issue with the italics not rendering

Reply:
done